### PR TITLE
Update dependencies and fix typescript errors

### PR DESCRIPTION
**Pull request by Khaledgarbaya**
## Todos

-   [x] Implemented feature
-   [ ] Feature with pending implementation

## Screenshots (if appropriate):

### CODE REVIEW

**Comment by Khaledgarbaya** [comment]
I keep hitting the rate limit for environment creation on the CI let's wait till tomorrow

File: src/lib/offline-api/validator/schema/fields-schema.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ const enforceDependency = function ({ valid, when, is }) {
     is: is, then: Joi.any().required(), otherwise: Joi.any().forbidden()
   }).error((errors) => {
     return errors.map((error) => {
+      debugger
```
Comment (phoebeschmidt) [inline src/lib/offline-api/validator/schema/fields-schema.ts]: remove pls :)

File: test/unit/lib/intent-validator/content-transform.spec.ts [hunk 2]
```diff
@@ -137,7 +137,7 @@ describe('Content transformation', function () {
         const validationErrors = await validateSteps(function up (migration) {
           migration.transformEntries(properties)
         })
-
+        debugger
```
Comment (phoebeschmidt) [inline test/unit/lib/intent-validator/content-transform.spec.ts]: this one too :)

File: src/lib/offline-api/validator/schema/fields-schema.ts (hunk 1 above)
Comment (Khaledgarbaya) [reply src/lib/offline-api/validator/schema/fields-schema.ts]: 🤦‍♂

**Pull request merged**